archives/20170614/webrtc-stats.html

```diff
@@ -3305,6 +3306,16 @@ <h4 id="dictionary-rtcdatachannelstats-members" resource="#dictionary-rtcdatacha
                   <code>RTCDatachannel</code>, i.e., not including headers or padding.
                 </p>
               </dd>
+
+              <dt>
+                <dfn data-dfn-for="rtcdatachannelstats" data-dfn-type="dfn" id="dom-rtcdatachannelstats-currentRoundTripTime" data-idl="" data-title="currentRoundTripTime"><code>currentRoundTripTime</code></dfn> of type <span class="idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the latest round trip time measured in seconds, exposed by [<cite><a class="bibref" href="">usrsctp - spinfo_srtt</a></cite>].
```

Comment:
The correct reference for spinfo_srtt is RFC 6458: https://tools.ietf.org/html/rfc6458#section-8.2.2